src/main/java/http_server/ResponseBuilder.java

```diff
@@ -0,0 +1,119 @@
+package http_server;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.LinkedHashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class ResponseBuilder {
+    private Response response;
+    private Request request;
+    private Router router;
+    private HashMap<String, Callback> methodCollection;
+    private String crlf = "\r\n";
```

Comment:
This is used a lot in other files, isn't it? Is there any opportunity to create a constant somewhere?